File: 2sem/lab6/lab6/Circle.cpp
#include "Circle.h"

Circle::Circle(float r) {
	_radius = r;
}

double Circle::GetSquare() {
	return 3.1415 * _radius * _radius;
}

double Circle::GetPerimeter() {
	return 3.1415 * 2 * _radius;
}

File: 2sem/lab6/lab6_sharp/Circle.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace lab6_sharp
{
    class Circle : Figure
    {
        private float _radius;

        public Circle(float r)
        {
            _radius = r;
        }

        public override double GetPerimeter()
        {
            return 3.1415 * _radius * _radius;
        }

        public override double GetSquare()
        {
            return 3.1415 * 2 * _radius;
        }
    }
}
